__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const request = require('supertest');
+const {app} = require("../app");
+const {categoryData, commentData, reviewData, userData} = require('../db/data/test-data/index')
+
+beforeEach(() => seed({categoryData, commentData, reviewData, userData}));
+afterAll(() => db.end());
+
+describe("GET", () => {
```

Comment:
you don't have any way of handling this right now? You will need to update your server to catch any invalid endpoint